packages/react-scripts/scripts/utils/verifyTypeScriptSetup.js

```diff
@@ -86,8 +67,44 @@ function verifyTypeScriptSetup() {
     process.exit(1);
   }
 
+  const compilerOptions = {
```

Comment:
Why was this object moved? Seems like unnecessary diff noise.

Reply:
It was done to add in the `parsedValue` properties, which is using the typescript compiler reference. Alternatively we could leave it in place and hard code the values from typescript. Here's an example:

```ts
    enum ModuleKind {
        None = 0,
        CommonJS = 1,
        AMD = 2,
        UMD = 3,
        System = 4,
        ES2015 = 5,
        ESNext = 6
    }
```

What do you think?

Reply:
Let's not hard code. I'm fine with this move.